AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -157,7 +159,23 @@ final class InitialSyncOperation: AsynchronousOperation {
         let items = syncQueryResult.items
         recordsReceived += UInt(items.count)
 
+        if lastSyncTime != nil {
+            modelSyncedPayload.isFullSync = false
+            modelSyncedPayload.isDeltaSync = true
+        } else {
+            modelSyncedPayload.isFullSync = true
+            modelSyncedPayload.isDeltaSync = false
+        }
```

Comment:
you can remove the `if/else` entirely:

```swift
modelSyncedPayload.isFullSync = lastSyncTime == nil
modelSyncedPayload.isDeltaSync = lastSyncTime != nil
```

Reply:
Thanks, will update it, but the `ModelSyncedResult` should not be dispatch in this class.

What is made clear in today's (Sep, 1st, 2020) meeting, Android and iOS both should update the implementation of event `modelSynced`